guides/doc-Administering_Project/master.adoc

```diff
@@ -60,7 +60,11 @@ include::common/assembly_using-foreman-webhooks.adoc[leveloffset=+1]
 
 include::common/assembly_searching-and-bookmarking.adoc[leveloffset=+1]
 
+:parent-context: {context}
```

Comment:
I wonder why we need this. Did you try without? Do we include a specific module twice and therefore we need to overwrite the `context` here?

Reply:
Yes, this is so that I can re-include the procedure on checking the status of Foreman services (`proc_retrieving-status-of-services.adoc`).